### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by zerolab**
This PR updates a number of dependencies, including sentry

Closes #92, #94, #96, #87

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,31 +1,34 @@
...
-raven==6.7.0
-whitenoise==3.3.1
+whitenoise==5.2.0
 brotlipy==0.7.0
 django-storages==1.6.6
 boto3==1.7.82
 django-redis==4.9.0
+sentry-sdk==1.0.0
+scout-apm==2.18.0
+fabric3==1.14.post1
```
Comment (Stormheg) [inline requirements.txt:19]: You've added this to `requirements-dev.txt` as well. Did you mean to add it here in the base requirements?

File: wagtailio/settings/base.py [hunk 2]
```diff
@@ -397,19 +392,44 @@
 }
 
 
-# Sentry
+# Sentry configuration.
+# See instructions on the intranet:
+# https://intranet.torchbox.com/delivering-projects/tech/starting-new-project/#sentry
```
Comment (Stormheg) [inline wagtailio/settings/base.py:397]: This is an internal Torchbox link. Probably best to remove it.

File: wagtailio/settings/base.py [hunk 3]
```diff
@@ -397,19 +392,44 @@
...
+    from sentry_sdk.utils import get_default_release
+
+    sentry_kwargs = {
+        "dsn": env["SENTRY_DSN"],
+        "integrations": [DjangoIntegration()],
+    }
+
+    # There's a chooser to toggle between environments at the top right corner on sentry.io
+    # Values are typically 'staging' or 'production' but can be set to anything else if needed.
+    # dokku config:set gosh SENTRY_ENVIRONMENT=staging
```
Comment (Stormheg) [inline wagtailio/settings/base.py:413]: `gosh` is Torchbox client app I think? Probably better to remove this line as wagtail.io is not hosted using dokku.

**Review by Stormheg — APPROVED** [review]
Thanks @zerolab! Wagtail.io needs this kind of love :heart: 
I've added some small comments for things I think can be removed.

Tested this on my machine with the new vagrant box and staging data and everything appears to work correctly. I'm not able to verify the sentry configuration but I trust it to be correct.

File: requirements.txt (hunk 1 above)
Comment (zerolab) [reply requirements.txt]: nice catch. borked rebase

**Pull request merged**